src/python/WMCore/WMBS/CreateWMBSBase.py

```diff
@@ -278,15 +278,15 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE TABLE wmbs_job (
              id           INTEGER       PRIMARY KEY AUTO_INCREMENT,
              jobgroup     INTEGER       NOT NULL,
-             name         VARCHAR(255),
+             name         VARCHAR(70),
              state        INTEGER       NOT NULL,
              state_time   INTEGER       NOT NULL,
              retry_count  INTEGER       DEFAULT 0,
              couch_record VARCHAR(255),
              location     INTEGER,
              outcome      INTEGER       DEFAULT 0,
-             cache_dir    VARCHAR(767)  DEFAULT 'None',
-             fwjr_path    VARCHAR(767),
+             cache_dir    VARCHAR(70)  DEFAULT 'None',
+             fwjr_path    VARCHAR(70),
```

Comment:
Dirk, you're correct, the problem with unique constraints not the fields by
attributes. I was not clear on this. Anyway, I had enough headache to
change the code that I would worry right away about mariadb tweaking.

If someone knows how to tweak maria DB setup please speak up. Otherwise I'll
come back to this issue later when time permit. So far my goal was to
get things running with WMCore changes.

On  0, Dirk Hufnagel <notifications@github.com> wrote:
> >               state        INTEGER       NOT NULL,
> >               state_time   INTEGER       NOT NULL,
> >               retry_count  INTEGER       DEFAULT 0,
> >               couch_record VARCHAR(255),
> >               location     INTEGER,
> >               outcome      INTEGER       DEFAULT 0,
> > -             cache_dir    VARCHAR(767)  DEFAULT 'None',
> > -             fwjr_path    VARCHAR(767),
> > +             cache_dir    VARCHAR(70)  DEFAULT 'None',
> > +             fwjr_path    VARCHAR(70),
> 
> MariaDB supports fields longer than 255. You have some problems with combined unique constraints, that's all. These columns CANNOT be that small, otherwise everything we do will break (we store very long strings there all the time).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6260/files#r39897595